lib/agent/actions/fileretrieval/storage.js

```diff
@@ -9,14 +9,16 @@ var join      = require('path').join,
 var logger   = common.logger;
     watching = false;
 
-var NAMESPACE = "fr/";
+var storage_path = join(common.system.paths.config, 'commands.db');
+storage.init('files', storage_path, function(err) {
```

Comment:
One of the purposes of init'ing things, or constructing objects, is that you can later use their reference for the context for which you set them up, so if you are init'ing for `files` here, you shouldn't be needing to use `files` again several times below